### ISSUE TITLE
Support unix streams

### ISSUE THREAD

**Issue by mikepqr** [mikepqr/commpy#1]
This is a bug
```
$ commpy <(echo 1) <(echo 2)
1

$ comm <(echo 1) <(echo 2)
1
        2
```

**Comment by mikepqr** [comment]
This is a bug
```
$ commpy <(echo 1) <(echo 2)
1

$ comm <(echo 1) <(echo 2)
1
        2
```

**Comment by mikepqr** [comment]
Fixed in 4a15a89dd9e2366d860543c9ec8fc731ac31b5f2.
